.gitignore

```diff
@@ -11,3 +11,7 @@
 .phpunit
 /phpunit.xml
 ###< symfony/phpunit-bridge ###
+
+###> lexik/jwt-authentication-bundle ###
+#%CONFIG_DIR%/jwt/*.pem
```

Comment:
what about adding `bin/generate-jwt` instead? and keep ignoring the pem files, which is what the bundle does by default AFAIK: https://github.com/symfony/recipes/blob/master/lexik/jwt-authentication-bundle/2.3/manifest.json#L16